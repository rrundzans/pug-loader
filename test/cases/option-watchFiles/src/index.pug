h1 Hello Pug!
include:code sample.scss