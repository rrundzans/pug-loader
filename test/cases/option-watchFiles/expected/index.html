<h1>Hello Pug!</h1><code class="language-">$color: red;
h1 {
  color: $color;
}</code>